pagopa/io-app
# feat: [IOBP-647] Latest used method pre-selection

## Short description
This PR implements the logic to pre-select a payment method from the user wallets that have been recently used and will be shown in the appropriate “Recently Used” list item

## List of changes proposed in this pull request
- Added an additional selector to filter the recently used payment method from the user wallet with the field `updateDate` as more recent;
- Added a pre-selection statement into the reducer when the action `paymentsGetPaymentUserMethodsAction.success` is fired to filter the most recent payment method.

## How to test
Start a new payment flow with the new payment section feature flag enabled;
When you start a payment, you should be able to see the most recently used payment method pre-selected

## Preview
https://github.com/pagopa/io-app/assets/34343582/38e455e8-3e10-450d-a360-8c4024d3a75b
